graphql/multiplication-connect/fetchUserData.ts

```diff
@@ -0,0 +1,37 @@
+import { gql } from "@apollo/client";
+
+/*- setup a mutation(?) to track the current user and add it to the table 
+  - on save button click the user's id should be written to the Hasura database
+  - setup this file to look like other gQL requests in this folder */
+
+// Loads the data for the currently logged in user
+export const FETCH_USER_MC_DATA = gql`
+  query fetchUserData($id: String = "wHZGkjB7CFf0cMrcGgI1pIHfke02") {
+    multiplicationConnectData(where: { id: { _eq: $id } }) {
```

Comment:
maybe better to start using the user that vithushan has set up instead of putting an additional column in your table.
you will need to do the following: 

this is a line of code he set up -- has quite a bit behind it:
const { user } = useAuth();

then every time you need to access the userId in one of your queries, you access it this way:
variables: {
      userId: user.uid,
    },

that also makes accessing the userId anywhere in the page easy.  you can use: user.Displayname